src/main/java/org/jitterbit/connector/dropbox/activities/ProcessFileActivity.java

```diff
@@ -216,6 +281,29 @@ private void init() {
     model.schemaContentType = org.jitterbit.connector.sdk.metadata.SchemaMetaData.SchemaContentType.XML;
     model.filename = obj.getObjectName() + ".xml";
 
+    model = new DiscoverableObjectModel();
```

Comment:
Add a comment explaining what this code is demoing. Perhaps indicate why the repetition (as it probably would not be something that one would do in practice, but instead have a method to factor out the code.)